src/linker/Linker.Dataflow/FlowAnnotations.cs

```diff
@@ -75,6 +75,16 @@ public DynamicallyAccessedMemberTypes GetTypeAnnotation (TypeDefinition type)
 			return GetAnnotations (type).TypeAnnotation;
 		}
 
+		public bool DoesMemberAccessRequireDynamicallyAccessedMembers (IMemberDefinition provider) =>
+			provider switch {
+				MethodDefinition method =>
+					GetAnnotations (method.DeclaringType).TryGetAnnotation (method, out var annotation) &&
+					(annotation.ParameterAnnotations != null || annotation.ReturnParameterAnnotation != DynamicallyAccessedMemberTypes.None),
+				FieldDefinition field =>
+					GetAnnotations (field.DeclaringType).TryGetAnnotation (field, out _),
+				_ => false
+			};
```

Comment:
In my change there are very similar methods added as well. Please make sure we keep only one copy of these :smile: